### ISSUE TITLE
Removal Safety Check language should be improved

### ISSUE THREAD

**Issue by mosefer** [NationalSecurityAgency/skills-service#1162]
Removal Safety Check states "Please type Delete Me to permanently remove the record" however this is unclear for screen reader users as there's no mention of a button click being necessary to enact the deletion. Language should include mention of needing to click the Yes, Please Remove button after typing Delete Me.

**Comment by sudo-may** [comment]
PR: https://github.com/NationalSecurityAgency/skills-service/pull/1345
